tests/ha/test_ha_node_health.py

```diff
@@ -171,80 +160,60 @@ def test_nodes_one_by_one_os_shutdown(self):
         Test to Check that correct node status is shown in Cortx CLI and REST when node goes down
         and comes back up(one by one, os shutdown)
         """
-        LOGGER.info(
-            "Started: Test to check node status one by one for all nodes with os shutdown.")
+        LOGGER.info("Started: Test to check node status one by one for all nodes with os shutdown.")
 
         LOGGER.info("Shutdown nodes one by one and check status.")
         for node in range(self.num_nodes):
             self.restored = False
             node_name = self.srvnode_list[node]
-            LOGGER.info("Shutting down {}".format(node_name))
             if self.setup_type == "HW":
-                LOGGER.debug(
-                    "HW: Need to disable stonith on the node before shutdown")
+                LOGGER.debug("HW: Need to disable stonith on the node before shutdown")
                 self.node_list[node].execute_cmd(
-                    common_cmds.PCS_RESOURCE_STONITH_CMD.format("disable", node+1),
-                    read_lines=True)
-            resp = self.ha_obj.host_safe_unsafe_power_off(
-                host=self.host_list[node],
-                node_obj=self.node_list[node],
-                is_safe=True)
-            assert_utils.assert_true(
-                resp, "Host has not shutdown yet.")
-
-            LOGGER.info(
-                "Check in cortxcli and REST that the status is changed for {} to Failed".format(node_name))
+                    common_cmds.PCS_RESOURCE_STONITH_CMD.format("disable", node+1), read_lines=True)
```

Comment:
is "node + 1" reqd?

Reply:
Not sure. I have just corrected the indent. May be @RajeshPratapSInghChouhan can suggest on this.

Reply:
Spaces around +.

Reply:
As discussed, since this was with CLI commands, and was written accordingly. Hence keeping as is.

Reply:
node+1 is string or you are doing addition here?
If addition then it should be node + 1.